src/engraving/rw/read206/read206.cpp

```diff
@@ -750,6 +750,7 @@ static void readStaff(Staff* staff, XmlReader& e, ReadContext& ctx)
 
 void Read206::readPart206(Part* part, XmlReader& e, ReadContext& ctx)
 {
+    part->setPreferSharpFlat(PreferSharpFlat::NONE);
```

Comment:
You should also add it to read302

Reply:
It looks like there is no equivalent in read302; it uses `read400::TRead::read(part, e, ctx);`.

Reply:
Exactly.  Strange but true.  I did test using a 362 score and the fix works.